src/platform-aws.c

```diff
@@ -85,16 +85,16 @@ struct ec2_platform_data {
 	},
 	{
 		.name = "trn1.32xlarge",
-		.default_protocol = "SENDRECV",
```

Comment:
unrelated change.

Reply:
This is a required change to avoid warnings. When using designated initializers, but not providing all initializers, a warning is emitted if the selected initializers do not appear in their declaration order. This was the case for these, and I've reordered them to put them in declaration order.

Reply:
Split to a separate commit for git history